src/sentry/tasks/post_process.py

```diff
@@ -595,6 +595,37 @@ def process_rules(job: PostProcessJob) -> None:
     return
 
 
+def process_code_mappings(job: PostProcessJob) -> None:
+    if job["is_reprocessed"]:
```

Comment:
What are reprocessed jobs? and why do we skip them?

Reply:
I noticed this logic in a lot of the other post process jobs. IIUC skipping reprocessed events means we don't attempt to derive code mappings from them the second time around.